src/hevm/test/test.hs

```diff
@@ -260,27 +260,28 @@ tests = testGroup "hevm"
             |]
         [Cex _] <- withSolvers Z3 1 $ \s -> checkAssert s [0x21] c (Just ("fun(uint256)", [AbiUIntType 256])) []
         putStrLn "expected counterexample found"
-     ,
+     -- ,
      -- TODO 0x22 is missing: "0x22: If you access a storage byte array that is incorrectly encoded."
      -- TODO below should NOT fail
-     testCase "pop-empty-array" $ do
```

Comment:
can we use `expectFail` here instead of commenting

Reply:
Because the test doesn't not fail but it loops during `interpret`, expectFail also never terminates

Reply:
I think we can use `ignoreTest` in this case instead. It's a bit nice to keep even ignored tests typechecking at least

Reply:
Oh nice! I'll do that!